.circleci/config.yml

```diff
@@ -35,4 +36,4 @@ jobs:
             # add tag for omg-js sha that this image is using
             gcloud container images add-tag \
             $TEST_RUNNER_TAG \
-            gcr.io/omisego-development/omg-js-testrunner:with-omg-js-$(cat ./OMG_JS_SHA)
+            gcr.io/omisego-development/omg-js-testrunner:with-omg-js-$(cat ./OMG_JS_SHA | head -c $GIT_SHA_SHORT_LENGTH)
```

Comment:
how about:

`gcr.io/omisego-development/omg-js-testrunner:$GIT_SHA_SHORT_LENGTH`

Reply:
I .... am lost. you want to tag with `7`??
`gcr.io/omisego-development/omg-js-testrunner:7`

Also, just to clarify, the docker image would be tagged with the testrunner sha. This `with-omg-js` would be the second tag for a quick linking back to the compatible omg-js it is using.

So if you look at the image, it will have 2 tags:
https://console.cloud.google.com/gcr/images/omisego-development/GLOBAL/omg-js-testrunner@sha256:c3a3b7af4f1f89b9b173cda7909baef762dd637443217c9859cca5770397d514/details?tab=info&project=omisego-development

Reply:
Here's an example so you can see what I mean: https://hub.docker.com/layers/omisego/child_chain/cb3659f/images/sha256-0fc92fadb81af07ce0286501fbddeef9195f702447c516ef7b49b502d24ff469?context=explore

This is the head of master for `elixir-omg`. The image's full name is `omisego/child_chain:cb3659f`

If you don't specify a registry then the default Docker behaviour is to use its own public registry. So `omisego/child_chain:cb3659f` is equivalent to `hub.docker.com/omisego/child_chain:cb3659f`

Using GCP this becomes `gcr.io/omisego-development/child_chain:cb3659f`.

I think I understand the second part - you want to denote compatibility with the `test-runner` and `omg-js`? Maybe we can just have this in the repo instead?